config/initializers/devise.rb

```diff
@@ -266,4 +266,5 @@
   config.allowed_otp_drift_seconds = 600
   config.max_login_attempts = 3 # max OTP login attemps, not devise strategies (e.g. pw auth)
   config.otp_length = 8
+  config.direct_otp_length = 8
```

Comment:
Do we want to make this 6 by default? Seems like that's what most other sites use. 8 might be too much for a user to type, and I'm not sure how much more secure than 6 it is.